torchvision/datasets/video_utils.py

```diff
@@ -329,29 +341,29 @@ def get_clip(self, idx):
             end_pts = clip_pts[-1].item()
             video, audio, info = read_video(video_path, start_pts, end_pts)
         else:
-            info = _probe_video_from_file(video_path)
-            video_fps = info.video_fps
+            _info = _probe_video_from_file(video_path)
+            video_fps = _info.video_fps
             audio_fps = None
 
-            video_start_pts = clip_pts[0].item()
-            video_end_pts = clip_pts[-1].item()
+            video_start_pts = int(clip_pts[0].item())
+            video_end_pts = int(clip_pts[-1].item())
```

Comment:
The output is already `int` if you call `.item()`. Unfortunately, `mypy` doesn't know that. In order to make the intent more clear here, i.e. "we want to make the static type checker happy", let's do

```suggestion
            from typing import cast
            video_start_pts = cast(int, clip_pts[0].item())
            video_end_pts = cast(int, clip_pts[-1].item())
```